src/main/java/app/model/boundary/BoundaryFactory.java

```diff
@@ -18,16 +18,23 @@ public static ArrayList<Boundary> make(SettingsObject obj)
         objects.add(create(obj.getType(), corner[2], corner[3], obj.getTeleportTo()));
         objects.add(create(obj.getType(), corner[3], corner[0], obj.getTeleportTo()));
         objects.removeAll(Collections.singleton(null));
+        if(obj.getType() == FurnitureType.TARGET)
+        {
+            double x = obj.getRect().getMinX() + obj.getRect().getWidth()/2;
+            double y = obj.getRect().getMinY() + obj.getRect().getHeight()/2;
+            objects.add(new Flag(new Vector(x, y), 10));
```

Comment:
and use Flag here instead of TargetBoundary?

Reply:
Flag is a type of furniture, target boundary is a boundary.  Furniture needs to be assigned boundaries

Reply:
I see, thanks